demo/site/src/documents/pages/blocks/SliderBlock.tsx

```diff
@@ -0,0 +1,75 @@
+import { PropsWithData, withPreview } from "@comet/cms-site";
+import { SliderBlockData } from "@src/blocks.generated";
+import { MediaBlock } from "@src/common/blocks/MediaBlock";
+import { RichTextBlock } from "@src/common/blocks/RichTextBlock";
+import { PageLayout } from "@src/layout/PageLayout";
+import styled from "styled-components";
+import SwiperCore from "swiper";
+import { Navigation } from "swiper/modules";
+import { Swiper, SwiperSlide } from "swiper/react";
+
+type SliderBlockProps = PropsWithData<SliderBlockData>;
+
+SwiperCore.use([Navigation]);
+
+export const SliderBlock = withPreview(
+    ({ data: { sliderList } }: SliderBlockProps) => {
+        return (
+            <Slider>
+                <SwiperContainer>
+                    <Swiper
+                        modules={[Navigation]}
+                        slidesPerView={3}
+                        spaceBetween={20}
+                        longSwipesRatio={0.1}
+                        threshold={3}
+                        allowTouchMove
+                        watchOverflow
+                        autoHeight
+                    >
+                        {sliderList.blocks.map((block) => (
+                            <SwiperSlide key={block.key}>
+                                <SliderItemBlockRoot>
+                                    <MediaWrapper>
+                                        <MediaBlock data={block.props.media} fill aspectRatio="16x9" sizes="50vw" />
+                                    </MediaWrapper>
+                                    <RichTextBlock data={block.props.text} />
+                                </SliderItemBlockRoot>
+                            </SwiperSlide>
+                        ))}
+                    </Swiper>
+                </SwiperContainer>
+            </Slider>
+        );
+    },
+    { label: "Slider" },
+);
+
+export const PageContentSliderBlock = (props: SliderBlockProps) => (
+    <PageLayout>
+        <SliderBlock {...props} />
+    </PageLayout>
+);
```

Comment:
The block isn't used outside of the page content, so there shouldn't be a page content variant, and the page layout should be used directly in SliderBlock

Reply:
True, fixed in 32740a6ad6eed35dfbff11e17301f520416cb4c6